operators/advanced_network/Dockerfile

```diff
@@ -75,18 +75,91 @@ RUN --mount=type=bind,source=${HOLOSCAN_DEB_LOCAL_PATH},target=/tmp/holoscan.deb
 FROM hsdk-${HOLOSCAN_DEB_SRC} AS hsdk-installed
 
 # ==============================================================
-# common-deps: common dependencies for the advanced network lib
+# base-deps: base dependencies for the advanced network lib
 # ==============================================================
-FROM hsdk-installed AS common-deps
+FROM hsdk-installed AS base-deps
 
 ARG TARGETARCH
-ARG DOCA_VERSION=2.8.0
 ARG CACHEBUST=1
 ARG DEBIAN_FRONTEND=noninteractive
 
 WORKDIR /opt
 
-# Configure the DOCA APT repository
+# APT installs for base dependencies (no DOCA/GPUNetIO packages)
+# - libibverbs-dev, librdmacm-dev: RDMA/ibverbs support for Mellanox NICs
+# - libmlx5-1: Mellanox ConnectX driver
+# - ibverbs-utils: utilities
+# - python3-dev: for building python bindings
+RUN apt-get update && apt-get install -y --no-install-recommends \
+        libibverbs-dev \
+        librdmacm-dev \
```

Comment:
I thought the `-dev` ones were only to build upstream DPDK, and we just needed the runtimes for the base (`libibverbs1`, `librdmacm1`, `rdma-core`)?

Reply:
becuase if I undestand correctly:
- runtime libs: `ibibverbs1 librdmacm1 rdma-core` are needed by all managers which are on top of base
- dev libs: `libibverbs-dev librdmacm-dev` are only used by DPDK? 


But totally fine to keep dev libs in base as well, just pointing it out.